tests/models/test_ensemble_base.py

```diff
@@ -0,0 +1,67 @@
+from typing import Dict
+
+import mock
+import torch
+import torch.nn as nn
+
+from fl4health.model_bases.ensemble_base import EnsembleAggregationMode, EnsembleModel
+from tests.test_utils.models_for_test import SmallCnn
+
+
+def test_forward_average_mode() -> None:
+    models: Dict[str, nn.Module] = {"model_0": SmallCnn(), "model_1": SmallCnn()}
+    em = EnsembleModel(models, EnsembleAggregationMode.AVERAGE)
```

Comment:
Maybe expand these variable names. em -> ensemble_model and ep -> ensemble_predictions